new-components/targets/git-clone/.env.git-clone

```diff
@@ -0,0 +1,6 @@
+SMITHY_COMPONENT_NAME=git-clone
```

Comment:
I'd rather this value is hardcoded in the a file with component variables, such as name, type etc.

Reply:
Injected in the main as an option